### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/nerdctl/commit/c98b159e50485e6c641f40c88606e55b6cb129d7#commitcomment-65143734

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/nerdctl/commit/c038f477e31fe55bcf7459f79648e8ac7eb0bbc8

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I intend to annotate this PR shortly. I'm happy to drop changes to files, drop specific terms, or split things as requested, although this may take some time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -648,7 +648,7 @@ Flags:
 ### :whale: nerdctl update
 Update configuration of one or more containers.
 
-Usage: `nerctl update [OPTIONS] CONTAINER [CONTAINER...]`
+Usage: `nerdctl update [OPTIONS] CONTAINER [CONTAINER...]`
```
Comment (jsoref) [inline README.md:651]: brand

File: README.md [hunk 2]
```diff
@@ -872,7 +872,7 @@ To see non-encrypted information, run `nerdctl image inspect --mode=native --pla
 Flags:
 -  `--recipient=<RECIPIENT>`      : Recipient of the image is the person who can decrypt (e.g., `jwe:mypubkey.pem`)
 -  `--dec-recipient=<RECIPIENT>`  : Recipient of the image; used only for PKCS7 and must be an x509 certificate
--  `--key=<KEY>[:<PWDESC>]`       : A secret key's filename and an optional password separated by colon, PWDDESC=<password>|pass:<password>|fd=<file descriptor>|filename
+-  `--key=<KEY>[:<PWDDESC>]`      : A secret key's filename and an optional password separated by colon, PWDDESC=<password>|pass:<password>|fd=<file descriptor>|filename
```
Comment (jsoref) [inline README.md:875]: This is for agreement w/ the description. Not sure about the preferred flavor

**Comment by jsoref** [comment]
At least part of https://github.com/containerd/nerdctl/runs/4995119640?check_suite_focus=true would be resolved by #757 

I'm not entirely sure about the rest of it.

I'm happy to squash the changes at the end, but often individual changes are controversial, so I prefer to leave them as distinct until they're deemed acceptable.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Comment by AkihiroSuda** [comment]
> I'm happy to squash the changes at the end, but often individual changes are controversial, so I prefer to leave them as distinct until they're deemed acceptable.

Please squash

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**